_posts/wookiemage/2016-2-3-Colins-Chapter-5.md

```diff
@@ -0,0 +1,13 @@
+---
+author: wookiemage
+layout: post
+title: "Colin's Chapter 5 Exercises
+"
+---
+  Here's the first exercise:
+  <iframe src="https://trinket.io/embed/python/39cbbd5989" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+  Here's the second:
+  <iframe src="https://trinket.io/embed/python/f584d23e33" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
I added the same condition to the max section too and saved it in Trinket. That should update the one in the post without my having to change the post, right?
:fire: